tests/tracing/test_tracing.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+from unittest.mock import patch
+
+
+from crewai import Agent, Task, Crew
+from crewai.utilities.events.listeners.tracing.trace_listener import (
+    TraceCollectionListener,
+)
+from crewai.utilities.events.listeners.tracing.trace_batch_manager import (
+    TraceBatchManager,
+)
+
+
+class TestTraceListenerSetup:
```

Comment:
Should we test the propageted paramtners? To make sure we are providing them corectly?